### PR TITLE
unregister services by using the maintenance mode

### PR DESCRIPTION
**Pull request by ghost**
@rebuy-de/prp-consul-java-client

### CODE REVIEW

**Comment by bjoernhaeuser** [comment]
:+1:

File: pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
         <mvn.release.version>2.5.3</mvn.release.version>
 
         <junit.version>4.11</junit.version>
-        <consul-api.version>1.1.8</consul-api.version>
+        <consul-api.version>1.2.0.REBUY</consul-api.version>
```
Comment (svenwltr) [inline pom.xml:19]: Do we want to jump to a new minor version? What if the next official release is a `1.1.9`? Then ours seems newer.

**Comment by svenwltr** [comment]
:+1:

File: pom.xml (hunk 1 above)
Comment (bjoernhaeuser) [inline pom.xml:19]: new Feature needs new minor, so this is okay :)

**Pull request merged**